save-common/src/jvmMain/kotlin/org/cqfn/save/core/utils/PlatformUtils.kt

```diff
@@ -12,6 +13,15 @@ actual typealias AtomicInt = java.util.concurrent.atomic.AtomicInteger
 
 actual typealias AtomicBoolean = java.util.concurrent.atomic.AtomicBoolean
 
+@Suppress("USE_DATA_CLASS")
+actual class GenericAtomicReference<T> actual constructor(valueToStore: T) {
+    private var value: T = valueToStore
```

Comment:
This should be `java.util.concurrent.atomic.AtomicReference`

Reply:
I made `private val holder: java.util.concurrent.atomic.AtomicReference`, seems like what we want.